pkg/webhook/nodenetworkconfigurationpolicy/nodenetworkconfigurationpolicy_suite_test.go

```diff
@@ -17,7 +18,8 @@ import (
 
 func TestUnit(t *testing.T) {
 	RegisterFailHandler(Fail)
-	RunSpecs(t, "NNCP Webhook Test Suite")
+	junitReporter := reporters.NewJUnitReporter("junit.webhook-webhook_suite_test.xml")
+	RunSpecsWithDefaultAndCustomReporters(t, "NNCP Webhook Test Suite", []Reporter{junitReporter})
```

Comment:
Why is this change needed?

Reply:
I noticed that the webhook tests didn't produce xml output, I'll remove this  change from this PR as it's not related anymore